File: APPTest/src/main/java/com/bihe0832/android/test/module/TestRouterFragment.java
package com.bihe0832.android.test.module;

import android.os.Bundle;

import com.bihe0832.android.lib.router.Routers;
import com.bihe0832.android.test.base.BaseTestFragment;
import com.bihe0832.android.test.base.OnTestItemClickListener;
import com.bihe0832.android.test.base.TestItem;

import java.util.ArrayList;
import java.util.List;

public class TestRouterFragment extends BaseTestFragment {

    public static TestRouterFragment newInstance() {

        Bundle args = new Bundle();
        TestRouterFragment fragment = new TestRouterFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public List<TestItem> getDataList() {
        List<TestItem> items = new ArrayList<TestItem>();
        items.add(new RouterItem("zixie://test"));
        items.add(new RouterItem("zixie://testhttp"));
        items.add(new RouterItem("zixie://feedback"));
        return items;
    }

    private class RouterItem extends TestItem{

        public RouterItem(final String url){
            super(url, new OnTestItemClickListener() {
                @Override
                public void onItemClick(TestItem testItem) {
                    Routers.open(getContext(),url);
                }
            });

        }
    }
}

File: APPTest/src/main/java/com/bihe0832/android/test/module/TestRouterFragment.kt
package com.bihe0832.android.test.module

import com.bihe0832.android.common.test.module.TestDebugRouterFragment
import com.bihe0832.android.lib.request.URLUtils

class TestRouterFragment : TestDebugRouterFragment() {

    override fun getRouterList(): ArrayList<TestDebugRouterFragment.RouterItem> {
        return ArrayList<TestDebugRouterFragment.RouterItem>().apply {
            add(RouterItem("zixie://test"))
            add(RouterItem("zixie://about"))
            add(RouterItem("zixie://testhttp"))
            add(RouterItem("zixie://zfeedback?url=https%3A%2F%2Fsupport.qq.com%2Fproduct%2F290858"))
            add(RouterItem("zixie://testlist"))
            add(RouterItem("zixie://testweb"))
            add(RouterItem("zixie://zweb?url=https%3A%2F%2Fblog.bihe0832.com"))
            add(RouterItem("zixie://zfeedback?url=https%3A%2F%2Fsupport.qq.com%2Fproduct%2F290858"))
            add(RouterItem("zixie://zweb?url=" + URLUtils.encode("https://play.google.com/store/apps/details?id=com.pubg.newstate")))
            add(RouterItem("zixie://zweb?url=" + URLUtils.encode("https://v.qq.com")))

        }
    }
}